apprise_api/api/templates/config.html

```diff
@@ -71,7 +71,7 @@ <h5>{% trans "Loaded Configuration" %}</h5>
   <div id="config" class="col s12">
     <p>
       {% blocktrans %}Define your configuration below:{% endblocktrans %}
-      <form id="addconfig" action="{% url "add" key %}" method="post">
+      <form id="addconfig" action="{{BASE_URL}}/add/{{key}}" method="post">
```

Comment:
This should be `{% url "add" key %}` or simply left as is (a relative path reference)